group-coordinator/src/main/java/org/apache/kafka/coordinator/group/share/ShareCoordinatorMetrics.java

```diff
@@ -76,10 +76,10 @@ public ShareCoordinatorMetrics(Metrics metrics) {
             "Average latency in write state per second."),
         new Avg());
     shareCoordinatorWriteLatencyTotalSensor.add(
-        metrics.metricName("write-latency-total",
+        metrics.metricName("write-latency-max",
             METRICS_GROUP,
-            "Cumulative total latency in write state per second."),
-        new CumulativeSum());
+            "Maximum total latency in write state per second."),
```

Comment:
From the KIP, "The maximum time taken for a share-group state write call, including the time to write to the share-group state topic."